runtime/appruntime/config/config.go

```diff
@@ -113,11 +113,15 @@ func (eak EncoreAuthKey) Copy() EncoreAuthKey {
 }
 
 type PubsubProvider struct {
-	NSQ *NSQProvider       `json:"nsq,omitempty"` // set if the provider is NSQ
-	GCP *GCPPubsubProvider `json:"gcp,omitempty"` // set if the provider is GCP
+	NSQ   *NSQProvider             `json:"nsq,omitempty"`   // set if the provider is NSQ
+	GCP   *GCPPubsubProvider       `json:"gcp,omitempty"`   // set if the provider is GCP
 	AWS *AWSPubsubProvider `json:"aws,omitempty"` // set if the provider is AWS
```

Comment:
Feels like you need to reformat here, as the AWS key hasn't been formatted inline with the other three